raiden/ui/cli.py

```diff
@@ -283,10 +291,20 @@ def run(ctx, **kwargs):
 
         app_.raiden.register_registry(app_.raiden.chain.default_registry.address)
 
+        domain_list = []
+        if kwargs['rpccorsdomain']:
+            if ',' in kwargs['rpccorsdomain']:
+                for domain in kwargs['rpccorsdomain'].split(','):
+                    domain = str("".join(["http://", domain, ":*/*"]))
```

Comment:
Actually even I thought about it long and hard but in the Issue that was created https://github.com/raiden-network/raiden/issues/420 there is a list of valid IPs from which to accept requests. So I assumed there must be some logic to it and I let remain that way. I could change it actually so that the entire domain can be filled by user.